content/docs/guides/saml/aad.md

```diff
@@ -63,20 +68,20 @@ The final step is to configure the **name identifier format**.
 
 ![User Attributes & Claims](/images/docs/reference/service/saml-aad/attributes-and-claims.png)
 
-1. In the **Manage User Claims** panel, expand the **Choose name identifier format** and select **Email address**.
+1. In the **Manage User Claims** panel, expand **Choose name identifier format** and select **Email address**.
 
 ![Manage Name Identifier Format](/images/docs/reference/service/saml-aad/name-identifier-format.png)
 
 1. Finally, click **Save** at the bottom of the **Manage User Claims** panel.
 
-**IMPORTANT:** Do not change the value of Name ID Format value once your users have started using Pulumi; not even switching its value between Email or Persistent.
+> **Important:** Do not change the value of Name ID Format value once your users have started using Pulumi---not even switching its value between Email or Persistent.
```

Comment:
Shouldn't we be using just two `--` dashes, which would then be replaced by an em dash? Or is the triple-dash somehow different/better?

Reply:
I think two dashes render an en dash.